content/shared/v3-processing-engine.md

```diff
@@ -0,0 +1 @@
+hello world! 
```

Comment:
Let's use `influxdb3-processing-engine.md` for this file path. It's winning out over `v3-`